msal/src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -1,332 +1,337 @@
-        }
-
-        private async Task<MsalTokenResponse> SendHttpMessageAsync(OAuth2Client client)
-        {
-            UriBuilder builder = new UriBuilder(AuthenticationRequestParameters.Authority.TokenEndpoint);
-            builder.AppendQueryParameters(AuthenticationRequestParameters.SliceParameters);
-            MsalTokenResponse msalTokenResponse =
-                await client
-                    .GetTokenAsync(builder.Uri,
-                        AuthenticationRequestParameters.RequestContext)
-                    .ConfigureAwait(false);
-
-            if (string.IsNullOrEmpty(msalTokenResponse.Scope))
-            {
-                msalTokenResponse.Scope = AuthenticationRequestParameters.Scope.AsSingleString();
-                AuthenticationRequestParameters.RequestContext.Logger.Info("ScopeSet was missing from the token response, so using developer provided scopes in the result");
-            }
-
-            return msalTokenResponse;
-        }
-
-        private void LogReturnedToken(AuthenticationResult result)
-        {
-            if (result.AccessToken != null)
-            {
-                AuthenticationRequestParameters.RequestContext.Logger.Info(
-                    string.Format(
-                        CultureInfo.InvariantCulture,
-                        "=== Token Acquisition finished successfully. An access token was returned with Expiration Time: {0} ===",
-                        result.ExpiresOn));
-            }
-        }
-    }
-}
\ No newline at end of file
+﻿//----------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;

```

Comment:
something got messed up with line endings.  can you fix?

Reply:
done